buildscripts/code-gen.sh

```diff
@@ -50,11 +50,14 @@ GROUPS_WITH_VERSIONS="$4"
 shift 4
 
 (
-  # To support running this script from anywhere, we have to first cd into this directory
-  # so we can install the tools.
-  #cd $(dirname "${0}")
-  cd vendor/k8s.io/code-generator/ 
-  go install ./cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
+# List of tools used for code generation
+TOOL=(client-gen deepcopy-gen lister-gen informer-gen)
+for i in ${TOOL[@]}; do
+  if [ ! -f "${GOPATH}/bin/$i" ]; then
+    echo "$i binary not found, installing at ${GOPATH}/bin"
+    go get k8s.io/code-generator/cmd/$i
```

Comment:
Since we are having vendor code for code-generator we can install it from vendor itself.

```suggestion
    go install ./vendor/k8s.io/code-generator/cmd/$i
```

Reply:
As discussed, undo these changes